specification/profiling.md

```diff
@@ -0,0 +1,169 @@
+# Profiling Conventions
+
+**Status**: [experimental](../README.md#versioning-and-status-of-the-specification)
+
+This document describes the specific behavior and conventions for Splunk
+agents that contain profiling features.
+
+- [Profiling Conventions](#profiling-conventions)
+    * [Activation](#activation)
+    * [Call Stack Sampling](#call-stack-sampling)
+        + [Call Stack Filtering](#call-stack-filtering)
+        + [Call Stack Span Association](#call-stack-span-association)
+        + [Call Stack Ingest](#call-stack-ingest)
+    * [ResourceLogs](#resourcelogs)
+        - [InstrumentationLibraryLogs](#instrumentationlibrarylogs)
+        - [LogRecord Common Attributes](#logrecord-common-attributes)
+        - [LogRecord Fields](#logrecord-fields)
+            * [Call Stack Format](#call-stack-format)
+- [References](#references)
+
+## Activation
+
+Agents may choose to activate or deactivate their profiler by default.
+A configuration setting MUST be provided to allow overriding the default: 
+
+* `SPLUNK_PROFILER_ENABLED` environment variable
+
+The default setting for `SPLUNK_PROFILER_ENABLED` MUST be clearly documented. 
+The agent SHOULD NOT allow changing the setting at runtime, and the initial 
```

Comment:
It simpler to say only the second part: that the agent will read the env var at initialization and will ignore any changes after the initialization. Notice that only the application itself could make "observable" changes to the env vars read by the agent (I assume that the agent is in the same process as the app).